### ISSUE TITLE
[Accessibility bug] html element does not have a [lang] attribute

### ISSUE THREAD

**Issue by alexsteeve** [topcoder-platform/Training-Challenges#381]
Steps to Reproduce: 
1. Open the main page (topcoder.com)
2. Open the DevTools to see the source code
3. Look the <html> tag
Actual Result: The html tag doesn't have the language
**Expected Result: ** <html lang="en"
Screenshots or screencast URL:

<img width="980" alt="Captura de Tela 2023-01-12 às 16 18 03" src="https://user-images.githubusercontent.com/42015163/212183239-b4514c79-af25-4448-8626-049a7656deb6.png">

Device/OS/Browser Information: MAC Ventura 13 Chrome

Accessibility Bug - 
<img width="336" alt="Captura de Tela 2023-01-12 às 16 19 05" src="https://user-images.githubusercontent.com/42015163/212183417-c505adfa-09c6-4379-ba1d-5cf759197f41.png">
<img width="761" alt="Captura de Tela 2023-01-12 às 16 19 29" src="https://user-images.githubusercontent.com/42015163/212183489-1ea12a32-b750-4e06-8bf4-834bf937a400.png">
